test/e2e/plain_provisioner_test.go

```diff
@@ -532,9 +532,67 @@ var _ = Describe("plain provisioner bundle", func() {
 				}).Should(BeTrue())
 			})
 		})
+
+		When("the bundle has a custom manifests directory", func() {
+			var (
+				bundle *rukpakv1alpha1.Bundle
+			)
+			BeforeEach(func() {
+				bundle = &rukpakv1alpha1.Bundle{
+					ObjectMeta: metav1.ObjectMeta{
+						GenerateName: "combo-git-custom-dir",
+					},
+					Spec: rukpakv1alpha1.BundleSpec{
+						ProvisionerClassName: plainProvisionerID,
+						Source: rukpakv1alpha1.BundleSource{
+							Type: rukpakv1alpha1.SourceTypeGit,
+							Git: &rukpakv1alpha1.GitSource{
+								Repository: "https://github.com/exdx/combo-bundle",
+								Directory:  "./dev/deploy",
+								Ref: rukpakv1alpha1.GitRef{
+									Branch: "main",
+								},
+							},
+						},
+					},
+				}
+				err := c.Create(ctx, bundle)
+				Expect(err).To(BeNil())
+			})
+
+			AfterEach(func() {
+				err := c.Delete(ctx, bundle)
+				Expect(err).To(BeNil())
+			})
+
+			It("Can create and unpack the bundle successfully", func() {
```

Comment:
supersuper nit: the rest of the e2e package seems to use lowercase for the first word in the It spec.